README.md

```diff
@@ -93,34 +54,44 @@ docker run -v /path/to/config/dir:/config \
            ghcr.io/maplibre/martin --config /config/config.yaml
 ```
 
-# Usage
+## Usage
 
-Martin requires at least one PostgreSQL [connection string](#postgresql-connection-string) or a [tile source file](#mbtile-and-pmtile-sources) as a command-line argument. A PG connection string can also be passed via the `DATABASE_URL` environment variable.
+### PostGIS sources
+Martin requires at least one PostgreSQL [connection string](https://maplibre.org/martin/PostgreSQL-Connection-String.html) or a [tile source file](https://maplibre.org/martin/MBTile-and-PMTile-Sources.html) as a command-line argument. A PG connection string can also be passed via the `DATABASE_URL` environment variable.
 
 ```shell
 martin postgresql://postgres@localhost/db
```

Comment:
should this also use `user@host` style of a link, same as above?

Reply:
The styles are same now